crown-court-proceeding/src/main/resources/schemas/common/apiFinancialAssessment.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "apiFinancialAssessment.json",
+  "type": "object",
+  "description": "Crown Court Overview",
+  "properties": {
+    "initResult": {
+      "type": "string",
+      "description": "Outcome of initial assessment"
+    },
+    "fassInitStatus": {
+      "description": "Assessment Status",
+      "existingJavaType": "uk.gov.justice.laa.crime.crowncourt.staticdata.enums.CurrentStatus"
+    },
+    "fullResult": {
+      "type": "string",
+      "description": "Outcome of full assessment"
+    },
+    "fassFullStatus": {
+      "description": "Assessment Status",
+      "existingJavaType": "uk.gov.justice.laa.crime.crowncourt.staticdata.enums.CurrentStatus"
+    },
+    "hardshipOverview": {
+      "type": "object",
+      "description": "Hardship over iew",
+      "$ref": "apiHardshipOverview.json"
+    }
+  },
+  "additionalProperties": false,
+  "required": ["initResult", "fassInitStatus", "fullResult", "fassFullStatus"]
```

Comment:
It's possible there might not be a `fullResult` and `fassFullStatus` if it's an initial assessment